https://bugzilla.gnome.org/show_bug.cgi?id=754154

--- origsrc/gdk-pixbuf-2.31.7/tests/Makefile.am	2015-08-24 15:27:17.000000000 -0500
+++ src/gdk-pixbuf-2.31.7/tests/Makefile.am	2015-09-01 17:09:21.800698800 -0500
@@ -18,10 +18,15 @@ LDADD = \
 
 noinst_PROGRAMS += 			\
 	pixbuf-read			\
-	pixbuf-lowmem			\
 	pixbuf-random			\
 	$(NULL)
 
+if OS_LINUX
+noinst_PROGRAMS += 			\
+	pixbuf-lowmem			\
+	$(NULL)
+endif
+
 test_programs = 			\
 	animation			\
 	cve-2015-4491			\
